package.json

```
@@ -75,7 +75,7 @@
   "dependencies": {
     "@textile/grpc-powergate-client": "2.3.1",
     "@textile/grpc-transport": "^0.4.0",
-    "ipfs-http-client": "^47.0.1",
+    "ipfs-http-client": "next",
```

No need to use the `next` tag any more.

```suggestion
    "ipfs-http-client": "^50.1.1",
```